Homework/HttpPractice/Services/Abstractions/IUserService.cs

```diff
@@ -0,0 +1,14 @@
+using HttpPractice.Dtos;
+using HttpPractice.Dtos.Responses;
+
+namespace HttpPractice.Services.Abstractions;
+
+public interface IUserService
+{
+    Task<UserDto> GetUserById(int id);
+    Task<UserResponse> CreateUser(string name, string job); 
+    Task<List<UserDto>> GetUsers();
+    Task<UserResponse> UpdateUser(string name, string job, int id);
```

Comment:
Maybe it's better to create special Dto for user and send it instead of several parameters